app/src/controllers/pages/selectors.js

```diff
@@ -148,11 +149,32 @@ export const createAlternativeQueryParametersSelector = ({
       sortingKey,
       namespace,
     }),
-    ({ [SIZE_KEY]: limit, [SORTING_KEY]: sort, [PAGE_KEY]: pageNumber, ...rest }) => {
+    ({ [SIZE_KEY]: limit, [SORTING_ORDER_KEY]: sort, [PAGE_KEY]: pageNumber, ...rest }) => {
       return { ...getAlternativePaginationAndSortParams(sort, limit, pageNumber), ...rest };
     },
   );
 
+export const createFilterQuerySelector = ({
+  defaultPagination,
+  defaultSorting,
+  sortingKey,
+  namespace,
+} = {}) =>
+  createSelector(
+    createQueryParametersSelector({
+      defaultPagination,
+      defaultSorting,
+      sortingKey,
+      namespace,
+    }),
+    ({ [SIZE_KEY]: limit, [SORTING_ORDER_KEY]: sort, [PAGE_KEY]: pageNumber, ...rest }) => {
+      return {
+        ...getAlternativePaginationAndSortParams(sort, limit, pageNumber),
+        search_criteria: getAppliedFilters(rest)?.search_criterias,
```

Comment:
Can we just extract the query via `createAlternativeQueryParametersSelector` and process its result to extract filters via `getAppliedFilters`?

Reply:
this is unlikely to simplify the task

Reply:
Not sure that unnecessary code duplications will simplify app maintenance.

Reply:
Done.